scripts/startup/README.md

```diff
@@ -0,0 +1,47 @@
+These scripts support starting the PANOPTES software automatically
+when the Linux computer that runs a PANOPTES starts or restarts.
+
+First we need to have the PANOPTES environement variables set in all
+the shells that are involved in running PANOPTES. The simplest
+solution is to set them globally on the system. For example,
+on Ubuntu:
+
+```bash
+sudo cp $POCS/scripts/startup/set_panoptes_env_vars.sh \
+        /etc/profile.d/
+sudo chmod +x /etc/profile.d/set_panoptes_env_vars.sh
+```
+
+This will then be executed when any user logs in. The exact means
+of doing this varies on different versions of Unix/Linux.
+
+Next, we need to arrange for `su_panoptes.sh` to be executed when
+the operating system launches. First copy it to /etc:
```

Comment:
Can we symlink this in case it changes?

Reply:
That is a security hole, as it allows the file to be changed without entering the password. For that reason, I prefer to copy it. The files that run as $PANUSER are not copied.

Reply:
Right, got it.